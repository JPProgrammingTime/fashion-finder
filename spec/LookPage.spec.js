test.todo('This is where tests will happen')
