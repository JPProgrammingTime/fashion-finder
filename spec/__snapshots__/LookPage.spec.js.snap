// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render() 1`] = `
<div>
  <div
    className="sc-htpNat hHQqNW"
  >
    <div
      className="sc-bwzfXH bUkOeZ"
      id="dismiss-button"
      onClick={[MockFunction]}
    >
      <svg
        fill="none"
        height="32"
        stroke="#EE8CA3"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth="2"
        viewBox="0 0 24 24"
        width="32"
        xmlns="http://www.w3.org/2000/svg"
      >
        <line
          x1="18"
          x2="6"
          y1="6"
          y2="18"
        />
        <line
          x1="6"
          x2="18"
          y1="6"
          y2="18"
        />
      </svg>
    </div>
    <div
      className="sc-bwzfXH bUkOeZ"
      id="like-button"
      onClick={[MockFunction]}
    >
      <svg
        fill="none"
        height="32"
        stroke="#A6BEFA"
        strokeLinecap="round"
        strokeLinejoin="round"
        strokeWidth="2"
        viewBox="0 0 24 24"
        width="32"
        xmlns="http://www.w3.org/2000/svg"
      >
        <polyline
          points="20 6 9 17 4 12"
        />
      </svg>
    </div>
  </div>
</div>
`;
